bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BridgeHandler.java

```diff
@@ -437,6 +450,24 @@ public List<Room> getRooms() throws InterruptedException {
         }
     }
 
+    public @Nullable List<Room> getRoomsWithCache() {
+        return roomCache.getValue();
+    }
+
+    public @Nullable String resolveRoomId(@Nullable String roomId) {
```

Comment:
@david-pace, do you think a unit tests for the new public function could make sense?

Reply:
You are right, it would make sense to add more unit tests for these new methods. I will add them as soon as I find some time.